### PR TITLE
docs: update commands with mayastor version

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: quickstart/preparing-the-cluster.md [hunk 1]
```diff
@@ -1,20 +1,8 @@
...
-
-This restriction will be removed in version 2.0
-{% endhint %}
-
-New MSN nodes can be provisioned within the cluster at any time after the initial deployment, as aggregate demands for capacity, performance and availability levels increase.
-
 ### Verify / Enable Huge Page Support
 
-_2MiB-sized_  Huge Pages must be supported and enabled on a MSN.  A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the Mayastor pod on each node, which should be verified thus:
+_2MiB-sized_  Huge Pages must be supported and enabled the nodes.  A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the Mayastor pod on each node, which should be verified thus:
```
Comment (avishnu) [inline quickstart/preparing-the-cluster.md]: on the nodes

Comment (anupriya0703) [reply quickstart/preparing-the-cluster.md]: updated the text.

**Review by avishnu — APPROVED** [review]
lgtm

File: quickstart/preparing-the-cluster.md [hunk 2]
```diff
@@ -1,20 +1,8 @@
...
-
-This restriction will be removed in version 2.0
-{% endhint %}
-
-New MSN nodes can be provisioned within the cluster at any time after the initial deployment, as aggregate demands for capacity, performance and availability levels increase.
-
 ### Verify / Enable Huge Page Support
 
-_2MiB-sized_  Huge Pages must be supported and enabled on a MSN.  A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the Mayastor pod on each node, which should be verified thus:
+_2MiB-sized_  Huge Pages must be supported and enabled on the nodes. A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the Mayastor pod on each node, which should be verified thus:
```
Comment (tiagolobocastro) [inline quickstart/preparing-the-cluster.md]: Shouldn't we still keep the "## Configure Mayastor Nodes" ?

```suggestion
_2MiB-sized_  Huge Pages must be supported and enabled on the mayastor storage nodes. A minimum number of 1024 such pages \(i.e. 2GiB total\) must be available _exclusively_ to the Mayastor pod on each node, which should be verified thus:
```

Comment (anupriya0703) [reply quickstart/preparing-the-cluster.md]: Updated the text. Thanks.

**Pull request merged**